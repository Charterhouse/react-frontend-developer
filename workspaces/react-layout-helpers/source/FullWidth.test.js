import React from 'react'
import { render } from 'react-testing-library'
import { FullWidth } from './'

describe('FullWidth', () => {
  it('has proper styling', () => {
    const { container } = render(<FullWidth />)
    expect(container).toMatchSnapshot()
  })
})
