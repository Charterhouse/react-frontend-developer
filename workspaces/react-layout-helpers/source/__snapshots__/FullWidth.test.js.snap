// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullWidth has proper styling 1`] = `
.emotion-0 {
  width: 100%;
  margin-top: 20px;
}

<div>
  <div
    class="emotion-0 emotion-1"
  />
</div>
`;
